bio_modules/dna_rna_module.py

```diff
@@ -0,0 +1,77 @@
+""" Module with auxiliary functions for bio_tools.py, run_dna_rna_tools(),
+filter_fastq() """
+
+
+dna = {"a", "t", "g", "c"}
+rna = {"a", "u", "g", "c"}
+rna_complements = {
+    "a": "u",
+    "u": "a",
+    "g": "c",
+    "c": "g",
+    "A": "U",
+    "U": "A",
+    "C": "G",
+    "G": "C",
+}
+dna_complements = {
+    "a": "t",
+    "t": "a",
+    "g": "c",
+    "c": "g",
+    "A": "T",
+    "T": "A",
+    "C": "G",
+    "G": "C",
+}
+
+
+def transcribe(seq: str) -> str:
+    for nucleotide in seq:
+        if nucleotide == "T":
+            seq = seq.replace("T", "U")
+        if nucleotide == "t":
+            seq = seq.replace("t", "u")
+    return seq
```

Comment:
Очень странно, ты итерируешься по нуклеотидам а меняешь строку. Вообще тебе тут просто не нужен цикл по нуклеотидам (зачем?). Да и проверка не нужна. Если нет - то и не заменится.
```suggestion
    return seq.replace("t", "u").replace("T", "U")
```